README.md

```diff
@@ -22,23 +22,45 @@ Read on to learn about the details!
 
 Install the app into your environment:
 
-> ```pip install django_components```
+```> pip install django_components```
 
 Then add the app into INSTALLED APPS in settings.py
 
-```
+```python
 INSTALLED_APPS = [
     ...,
     "django_components",
     ...
 ]
 ```
 
+Modify `TEMPLATES` section of settings.py as follows:
+- Remove `'APP_DIRS': True,`
+- add `loaders` to `OPTIONS` list and set it to following value:
+```python
+TEMPLATES = [
+    {
+        ...,
+        'OPTIONS': {
+            'context_processors': [
+                ...
+            ],
+            'loaders':[(
+                'django.template.loaders.cached.Loader', [
+                    'django.template.loaders.filesystem.Loader',
+                    'django.template.loaders.app_directories.Loader',
+                    'django_components.template_loader.Loader',
```

Comment:
This is the only line that's needed right? I have an idea of how to make this easier to follow, but I can fix that in a separate commit.

Reply:
Refer to this section https://docs.djangoproject.com/en/3.2/ref/templates/api/#django.template.loaders.cached.Loader